Show more/show less button on comment

drillprop opened the pull request:
Fix #2564 
Designs: 
https://www.figma.com/file/sMydmZJlzhK91FTIPOd7D2/Video-page?node-id=3677%3A265878

Be aware that the look of the show more button differs from the design. This won't be an issue once https://github.com/Joystream/atlas/issues/2463 is done

I didn't handle [exceeding characters limit story](https://www.figma.com/file/sMydmZJlzhK91FTIPOd7D2/?node-id=3677%3A254185). I'll create a separate issue just for that.

drillprop committed:
add show more button on comments

drillprop committed:
show only one empty line between paragraphs

rafalpawlow commented on packages/atlas/src/components/_comments/CommentBody/CommentBody.tsx:
```diff
@@ -0,0 +1,48 @@
+
+import { ExpandButton, StyledCommentText } from './CommentBody.styles'
+
+export const CommentBody = ({ children }: { children?: string }) => {
```
Can't we just add proper type here?
```suggestion
export const CommentBody: React.FC = ({ children }) => {
```

drillprop replied:
I want to be able to pass **only** string here, not React.ReactNode, that's why I wrote this.

rafalpawlow reviewed (changes requested):
1. there is wrong spacing between author and deleted comment. Can we fix this as part of this PR? We have `8px` but in figma it's `12px`
![Zrzut ekranu 2022-05-10 o 13 37 37](https://user-images.githubusercontent.com/13706246/167620494-1b645fc9-29a9-40b4-9306-3ce3c338d9ba.png)

2. I've noticed that Show more button has wrong color but I assume it will be fixed here https://github.com/Joystream/atlas/pull/2637 .

drillprop committed:
cr fixes

drillprop commented:
> 1. there is wrong spacing between author and deleted comment. Can we fix this as part of this PR? We have `8px` but in figma it's `12px`

Fixed

> 2. I've noticed that Show more button has wrong color but I assume it will be fixed here https://github.com/Joystream/atlas/pull/2637 . 

correct

drillprop committed:
add word break
